func/omp/custom_reduce.cpp

```diff
@@ -7,6 +7,9 @@ struct complex_t
     int imag;
 };
 
+// WARNING: custom reductions are not supported in a distributed Faasm
```

Comment:
Are they not?

Reply:
Unfortunately only reversible functions are supported, so we can't accept arbitrary user-defined reduction operations.